ui-components/src/locales/general.json

```diff
@@ -1523,6 +1523,19 @@
   "eligibility.accessibility.title": "Accessibility Features",
   "eligibility.accessibility.visual": "Visual",
   "eligibility.accessibility.wheelchairRamp": "Wheelchair Ramp",
+  "eligibility.accessibility.inUnitWasherDryer2": "In-unit washer/dryer",
```

Comment:
We should alphabetize these, and then I'm curious as to why we have some strings with the `2` label? If we aren't using the old ones, can we just remove them?